### ISSUE TITLE
New Relationship Field properties do not show important info

### ISSUE THREAD

**Issue by john-thomas-dotcms** [dotCMS/core#18706]
As a content architect, I want to see details of existing relationship fields in order to manage my content and content architecture.

### Describe the solution you'd like

The Relationships Field content type field properties popup needs to show the following additional information about each relationship field:

* The actual direction of a 1-N cardinality relationship.
    * Right now, both sides of a 1-N relationship show the relationship as "One to Many".  This means that you can't tell which of the Content Types is the parent, and which is the child.
    * The cardinality should show as "One to Many" from the parent Content Type, and should show as "Many to One" from the child Content Type - regardless of which Content Type the relationship was first created from.
* Which side of a relationship is considered the parent or child.
    * This can make a difference in some cases, and currently there's no way to tell.
* Which content type field in the related Content Type that the Relationship Field is connected to.
    * If you have 2 or more Relationships Fields between the same 2 Content Types, you can't tell which is connected to which, because the variable name of the Relationship Field in the connected Content Type is not shown.

## Additional Context

This information was (and is still) available with Legacy relationships, but is not available with the new Relationships Fields, so it may discourage customers from upgrading their Legacy relationships.

## Acceptance Criteria
1. Display all the information in the Relationships Field properties popup.
2. Work in all the supported browsers (don't forget IE11 and iPad)
3. Multilanguage
4. Unit test

**Comment by john-thomas-dotcms** [comment]
I'm trying to explain the limit of parent-child relationships to a customer, but the customer has no way to tell what direction the relationship goes.

Support ticket: https://dotcms.zendesk.com/agent/tickets/103321

This problem is further aggravated by the fact that there is a significant limitation on how relationships can be used, because you can only perform a Lucene query against a relationships field from the parent side.  But there's no way to find out which is the parent - in fact, which side of the relationship is the parent and which is the child aren't even displayed when creating the new relationship.  So, customers have no way to tell whether a relationship will be limited or not - other than trying it and having it fail.

I'm upgrading this issue from an enhancement request to a bug - because, honestly, the information that's missing from this screen used to be available - so we need to fix this just to get back to where we can see the same information that was available before.

**Comment by wezell** [comment]
We need to add a "searchable by: label on the field screen if the relationship specifies the full relationship, e.g.

If you look at the blog <-> blog comment relationship - the blog comments can be searched via lucene - we need to expose this fact in the UI.

+Blog.blogComment:*

**Comment by rahulbpatel** [comment]
I'm not sure if this is being captured above.  Assume a one-to-many relationship is created between Blog and Blog Author with the Blog being the parent.  The relationship is added to both content types.  On Blog Author, the user should _not_ be presented a "User Searchable" option for the BlogAuthor.blog relationship field.  If I'm understanding correctly, a search field for Blog gets displayed in Content > Search but it will not actually work.  This is the situation we're in right now.

**Comment by john-thomas-dotcms** [comment]
Yes, @rahulbpatel you're correct - the plan is to provide information on how to search using a Lucene query from the parent side of the relationship, but not the child side of the relationship.

The ability to perform searches on relationships using Lucene queries is fairly new to dotCMS (as of the 5.x series).  But ultimately, we're only able to allow these searches in one direction.  However, from a customer perspective, what matters is not that one side is the parent and one is the child - what matters is whether you can perform these searches.

So, what this means is that, after we make this change, when you look at the relationships field in the content type, you'll be able to tell both if it's searchable using a Lucene query, and what specific field name to use when performing the search.

**Comment by rahulbpatel** [comment]
I'm not so worried about "Lucene queries" per se but rather it's very confusing if the UI makes you think it's available.

**Comment by fmontes** [comment]
After a call with @john-thomas-dotcms 

1. Show the related fields (not only the content types)
2. Add something to show how to make a Lucene query from the parent
3. Show what's the one and what's the many

**Comment by fmontes** [comment]
```
relationships: {
    cardinality: 1,
    velocityVar: "Person.children",
},
```

- If the field have a `.` in the `velocityVar` it means I'm the children of the relationship and if not it is the parent.

**Comment by fmontes** [comment]
Some mocks: https://docs.google.com/drawings/d/1Hu7mgy7KLta8bo60xdb1MDsJxoB3WePb4mQTC6-Jpas/edit

**Comment by fmontes** [comment]
MOCK:

![image](https://user-images.githubusercontent.com/751424/106940725-83399680-66e7-11eb-9ecb-9c7eb5c0d488.png)

**Comment by john-thomas-dotcms** [comment]
Nice @fmontes.  What would it look like from the other side - would it just be missing that bottom box?

**Comment by fmontes** [comment]
@john-thomas-dotcms yes, lose the query and the purple dot will move to the current content type we are editing

**Comment by fmontes** [comment]
The UI have some problems

<img width="812" alt="Screen Shot 2021-02-25 at 10 12 42 AM" src="https://user-images.githubusercontent.com/751424/109181961-1d728480-7752-11eb-844c-20a31399c0b5.png">

**Comment by bryanboza** [comment]
Fixed, tested on release-21.03 // Postgres // FF
